src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -316,6 +327,10 @@ const emptyStateStyles = {
   }),
 };
 
+function getFieldByLabel(fields: Array<SelectableValue<DetectedField>>, fieldLabel: string | undefined) {
```

Comment:
We should really not define functions with required arguments that can be undefined. Rather, we should not call this function if `fieldLabel` is undefined.

Random, but, for example, this would return a field with undefined label, if both things happen at the same time. In any case, even without running conditions, an undefined required parameter is not a good funciton signature.

Reply:
True! Fun fact, and this should not be an excuse, I used the "Refactor -> Extract as function" functionality for this.